app/components/nautilus/client.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Nautilus
+  # HTTParty client for Nautilus that takes an article path
+  # and get's the text of the article
+  class Client < GenericClient
```

Comment:
Assumes too much for instance variable '@response_body' (InstanceVariableAssumption)